### PR TITLE
Use older trusty image on Travis due to broken build

### LINKED ISSUE #11203: AMP runtime build broken on Travis due to new trusty image
Travis tracking bug: https://github.com/travis-ci/travis-ci/issues/8315
Announcement: https://blog.travis-ci.com/2017-08-29-trusty-image-updates
Workaround: Add `group: deprecated-2017Q3` to `.travis.yml`

### PR DESCRIPTION
**Pull request by rsimha-amp**
Temporarily fixes #11203, until the root causes are investigated

### CODE REVIEW

**Comment by rsimha-amp** [comment]
This fixed our build errors, but we're still seeing errors due to missing plugins, even though `yarn.lock` is up to date. See https://travis-ci.org/ampproject/amphtml/jobs/273011615

I'll raise this with the Travis team via https://github.com/travis-ci/travis-ci/issues/8315

**Comment by rsimha-amp** [comment]
Disabling caching did the trick. I'll follow up with a separate PR that fixes issues with the new Travis image.

File: .travis.yml [hunk 1]
```diff
@@ -54,10 +55,11 @@ matrix:
...
         jwt:
           secure: "Wze0F0vGL0UcxryOx1n/vcuD5LIMGyR+69Nc6IWLoRvZBbbIpFwVFhDE6rE9ranIXiA2Hc684N4sV8ASfNDF8RRSB+jyLov159qwgji2rBxIfQ/4kuDV2vYoAJvYMz8m42kwx5FV2VV9awqMMt8mwU3wYIrKIaVCxB34uV86KIlDlbrHxt17Bm5EIiUmwi9r1AAnW/63vVRUN264D77oB4j9UQ759PfD6BDwEt54O87KurNIaLseNCr1IvzfL8veEsZ3uTbLC1GtgHfR4IGgkS2YyN2QIk06VZWeRDEOalS3RcY0nDkbCmBywxIGObnrpEMzOpjBiOb2fxLoLvvpjlla5W84zJGfWE6q4T9IvkyHuDJE+sft5B+arjMIeA6PIeUhKdV27+6qqDEf7fILZ/U/Ekn9ds4zSV8hekAZPUyyPncOeyWppCIJ8sOeCrsebkRjH1BoX/d+FE+nP0bN/XkBpIi/nManx5FyS/kqjQWGKmvsFQfEWlSUaZi7XtEQEjvBizRkzvpJanSDaoiTDS2Keulmwii3XRId51FuGtnfDZFeggLaMTKGfBX9DlPkccwYAZe6vPNfYk1pNgEj6AtnifEhYVEO+aAuWhEnJ86od+1wDOL/h+a2XY6h8/gFBywsD95p7sXPfdVDCKgwagiBo+Hw5MNjztVF7lszg1A="
-cache:
-  yarn: true
-  directories:
-  - node_modules
-  - .gem
-  pip: true
-  bundler: true
+# Temporarily disabling caching due to a new image that broke us. See #11204
```
Comment (erwinmombay) [inline .travis.yml]: why did we have to do this? couldn't we just blow the cache manually?

**Comment by rsimha-amp** [comment]
I did this based on the suggestion at https://github.com/travis-ci/travis-ci/issues/8315#issuecomment-327888111. I didn't know it was possible to maually clear the cache without having to disable caching. 

I'll try to do so now, and then revert this change to see if all the errors go away.

File: .travis.yml (hunk 1 above)
Comment (rsimha-amp) [reply .travis.yml]: I did this based on the suggestion at travis-ci/travis-ci#8315 (comment). I didn't know it was possible to maually clear the cache without having to disable caching.

I'll try to do so now, and then revert this change to see if all the errors go away.

Comment (rsimha-amp) [reply .travis.yml]: Sent you #11206 for review.

**Pull request merged**